src/Blueprint/Blueprint/Cmdlets/BlueprintDefinition/SetAzureRMBlueprint.cs

```diff
@@ -0,0 +1,81 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Blueprint.Common;
+using Microsoft.Azure.Commands.Blueprint.Models;
+using System;
+using System.Management.Automation;
+using System.Text.RegularExpressions;
+using static Microsoft.Azure.Commands.Blueprint.Common.BlueprintConstants;
+
+namespace Microsoft.Azure.Commands.Blueprint.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Set, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "Blueprint", SupportsShouldProcess = true, DefaultParameterSetName = ParameterSetNames.UpdateBlueprintBySubscription), OutputType(typeof(PSBlueprint))]
+    public class SetAzureRMBlueprint : BlueprintDefinitionCmdletBase
```

Comment:
@filizt same comment about adding `-ResourceId` and `-InputObject` parameters

Reply:
@filizt are you able to make this change?

Reply:
@cormacpayne Not at this time. We'll update the cmdlets to have these parameters in the next preview release.